ci/before_install.sh

```diff
@@ -12,7 +12,11 @@ elif [[ ${TRAVIS_OS_NAME} == "windows" ]]; then
     echo "Before install: Windows OS"
 elif [[ ${TRAVIS_OS_NAME} == "linux" ]]; then
     echo "Before install: Linux"
+    sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ xenial main'
```

Comment:
Use bionic repo and add release keys
```suggestion
    wget -O- -nc https://dl.winehq.org/wine-builds/winehq.key | sudo apt-key add -
    sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ bionic main'
```

Reply:
Travis is actually using Xenial as the most up to date image even though bionic is specified. So need to change the travis config and obviously amend the wine repo used, we will add to this commit with the relevant changes.